tests/devices/ed/tea_bar_review_regression_test.py

```
@@ -0,0 +1,67 @@
+
+@pytest.mark.parametrize("reported_keep_warm_time", [None, 12.0])
+def test_tea_bar_keep_warm_uses_device_default_duration(
+    reported_keep_warm_time: float | None,
+) -> None:
+    """Do not send an explicit duration for the appliance's 12-hour default."""
+    tea_bar = MideaEDDevice(
+        name="Tea Bar",
+        device_id=2,
+        ip_address="192.0.2.1",
+        port=6444,
+        token="AA",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token"

(S106)